docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.install

```diff
@@ -246,3 +246,37 @@ function hs_paragraph_types_update_10006() {
     }
   }
 }
+
+/**
+ * Moves drag & drop action for paragraphs from 3 dot menu to paragraph toolbar.
+ */
+function hs_paragraph_types_update_10007() {
```

Comment:
Is this needed ? Does the `drush config import --partial` not import the relevant changes from config sync?

Reply:
This one is needed because those configs are ignored and need to be installed manually. See https://github.com/SU-HSDO/suhumsci/blob/develop/config/default/config_ignore.settings.yml